infra/helper.py

```diff
@@ -249,6 +251,11 @@ def _get_work_dir(project_name=''):
   return os.path.join(BUILD_DIR, 'work', project_name)
 
 
+def _add_architecture_args(parser, choices=('x86_64', 'i386')):
+  """Add common architecture args."""
+  parser.add_argument('--architecture', default='x86_64', choices=choices)
```

Comment:
maybe just --arch

Reply:
I think architecture is more descriptive and wouldn't "arch" be an anomaly in CF and OSS-Fuzz where we pick verbose names?